tests/test_e2ee_utils.py

```
@@ -0,0 +1,347 @@
+@patch("mmrelay.paths.resolve_all_paths")
+def test_credentials_found_in_legacy_location(
+    mock_resolve_all_paths, mock_exists, e2ee_test_config
+) -> None:
+    """
+    Test credential detection when credentials are found in legacy location (lines 115-122).
+
+    This test verifies that when credentials are not found in primary location
+    but exist in a legacy location, get_e2ee_status correctly marks
+    credentials as available and stops searching after first match.
+    """
+    temp_dir, config_path, credentials_path, base_config = e2ee_test_config
```

Ruff (0.14.14)

[warning] Unpacked variable `temp_dir` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)